pages/api/teams/[slug]/members.ts

```diff
@@ -154,6 +159,10 @@ const handlePATCH = async (req: NextApiRequest, res: NextApiResponse) => {
     req.body as { memberId: string; role: Role }
   );
 
+  const getMemberData = await throwIfCannotGetMember(req, memberId);
+
+  throwIfNotOwnerOnOwner(teamMember.role, getMemberData.role);
```

Comment:
The function `validateUpdateRole` is performing the same operation as `throwIfNotOwnerOnOwner`, and the function name is `throwIfNotOwnerOnOwner` a bit vague.
Can we use the same pattern of rbac for delete as well?